x/gamm/keeper/keeper_test.go

```diff
@@ -39,16 +39,14 @@ func (suite *KeeperTestSuite) prepareCustomBalancerPool(
 	balances sdk.Coins,
 	poolAssets []balancertypes.PoolAsset,
 	poolParams balancer.PoolParams,
-) uint64 {
+) (uint64, error) {
 	suite.fundAllAccountsWith(balances)
 
 	poolID, err := suite.App.PoolManagerKeeper.CreatePool(
 		suite.Ctx,
 		balancer.NewMsgCreateBalancerPool(suite.TestAccs[0], poolParams, poolAssets, ""),
 	)
-	suite.Require().NoError(err)
-
-	return poolID
+	return poolID, err
```

Comment:
Why do we need to bring `suite.Require().NoError(err)` to outside of this func for any purpose?

Reply:
In issue #3849 there is suggest with corner cases 0 <= exitFee < 1
however cannot create pool with exitFee > 0. So i want to get that error outside

Above pysel says that "I could be wrong, but as far as I know, the exit fee was recently removed from the osmosis pool through the administration. I don't know if the admin's decision has been reflected. reflected in the protocol or not, but if so, I think we should delete this field poolExitFee here".
And I removed testcase poolExitFee > 0